docs/blog/2024-07-25-nx-19-5-update.md

```diff
@@ -0,0 +1,117 @@
+---
+title: Nx 19.5 Update!!
+slug: 'nx-19-5-update'
+authors: ['Zack DeRose']
+cover_image: '/blog/images/2024-07-25/nx-19-5-thumbnail.png'
+tags: [nx, release]
+---
+
+## NEW: Pattern Support for `targetDefaults`
+
+We've added Pattern Support for `targetDefaults` in your `nx.json` file.
+
+Generally speaking, `targetDefaults` are a way for you to specify default configurations for your workspace. For example, if for your workspace, your `test` tasks always depend on your `build` task to have run before the `test`s can run, you'd add this as a `targetDefault`:
```

Comment:
> way for you to specify default configurations for your workspace..

Yep, to reduce repetitive configuration and to setup a task pipeline. We should include links in the text to

- https://nx.dev/recipes/running-tasks/reduce-repetitive-configuration#reduce-configuration-with-targetdefaults
- https://nx.dev/features/run-tasks#defining-a-task-pipeline